### PR TITLE
Add Shape control to BBH

### PR DESCRIPTION
**Pull request by knelli2**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml [hunk 1]
```diff
@@ -332,6 +310,38 @@ ControlSystems:
...
+    TimescaleTuner:
+      InitialTimescales: 0.2
+      MinTimescale: 1.0e-2
+      MaxTimescale: 10.0
+      IncreaseThreshold: 2.5e-4
+      DecreaseThreshold: 1.0e-3
+      IncreaseFactor: 1.01
+      DecreaseFactor: 0.98
+    ControlError:
+  ShapeB:
```
Comment (nilsvu) [inline tests/InputFiles/GeneralizedHarmonic/BinaryBlackHole.yaml]: Maybe use YAML anchors to set the same settings for both A and B?

**Review by nilsvu — DISMISSED** [review]
Does this need some more testing or should we merge and then test more?

**Comment by nilsvu** [comment]
gcc-9 isn't happy

**Comment by knelli2** [comment]
Merge and test later. People can always turn shape control off from the input file if it's causing a lot of issues.

**Review by nilsvu — APPROVED** [review]
MacOS CI failed for some unrelated reason

**Pull request merged**